### PR TITLE
Update Task and Enum for Task

### PR DESCRIPTION
**Pull request by ChanWeiJie**

### CODE REVIEW

File: src/main/java/manageezpz/model/task/Priority.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package manageezpz.model.task;
+
+public enum Priority {
```
Comment (dannytayjy) [inline src/main/java/manageezpz/model/task/Priority.java:3]: Good to separate out enum class and put the priority values inside

File: src/main/java/manageezpz/logic/parser/FindTaskCommandParser.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public FindCommand parse(String args) throws ParseException {
         Prefix taskType = getPrefix(argMultiMapTaskType);
         List<String> descriptions = getDescriptions(argMultiMapProperties);
         Date date = getTaskDate(argMultiMapProperties);
-        Task.Priority priority = getTaskPriority(argMultiMapProperties);
+        Priority priority = getTaskPriority(argMultiMapProperties);
```
Comment (alfredkohhh) [inline src/main/java/manageezpz/logic/parser/FindTaskCommandParser.java:48]: Thanks for refactoring according to our discussion !

File: src/test/java/manageezpz/logic/parser/FindTaskCommandParserTest.java [hunk 3]
```diff
@@ -188,7 +188,7 @@ void findTaskCommandParser_multipleArguments_findCommand() {
                 PREFIX_PRIORITY.toString(), VALID_PRIORITY, PREFIX_ASSIGNEES.toString(), VALID_NAME_AMY,
                 PREFIX_IS_MARKED.toString(), VALID_BOOLEAN);
         TaskMultiplePredicate expectedPredicate = new TaskMultiplePredicate(PREFIX_DEADLINE, LIST_DESCRIPTIONS,
-                new Date(VALID_DATE), Task.Priority.valueOf(VALID_PRIORITY), VALID_NAME_AMY,
+                new Date(VALID_DATE), Priority.valueOf(VALID_PRIORITY), VALID_NAME_AMY,
```
Comment (alfredkohhh) [inline src/test/java/manageezpz/logic/parser/FindTaskCommandParserTest.java:191]: Thanks for updating the test as well !

**Review by alfredkohhh — APPROVED** [review]
LGTM! Thanks @ChanWeiJie

**Pull request merged**